app/models/Event.js

```diff
@@ -47,27 +47,25 @@ const eventSchema = new Schema({
       }
     }
   },
-  rsvp: {
-    yes: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    no: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    mayBe: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    }
-  },
+
+  rsvp_yes: [{
```

Comment:
why are you not following the convention here? 
if you see the file all the variable in schema are camelcase.

Reply:
@devesh-verma my apologies, i did not considered that before, just working on it 
anything else I need to work on plz let me know